addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -841,6 +851,43 @@ options.Class.include({
             },
         });
     },
+    /**
+     * Caches the rpc.
+     *
+     * @override
+     */
+    async _rpc() {
+        const cacheId = JSON.stringify(...arguments);
+        if (!this._rpcCache[cacheId]) {
+            this._rpcCache[cacheId] = this._super(...arguments);
+        }
+        return this._rpcCache[cacheId];
+    },
+
+    /**
+     * @override
+     */
+    async _computeWidgetVisibility(widgetName, params) {
+        const _super = this._super.bind(this);
+        if (
+            [
+                "footer_language_selector_label_opt",
+                "footer_language_selector_opt",
+                "header_language_selector_label_opt",
+                "header_language_selector_opt",
+            ].includes(widgetName)
+        ) {
+            this._languages = await this._rpc(
+                { 
+                    route: "/website/get_languages" 
+                }
+            );
+            if (this._languages.length === 1) {
+                return false;
+            }
```

Comment:
```suggestion
            return this._languages.length > 1;
```

Reply:
It's not exactly the same. If this._languages.length === 1 I have to return false otherwise _super(...arguments)